src/classes/groundplaces.ts

```diff
@@ -27,23 +40,34 @@ export class GroundPlaces {
   /**
    * @description Create the stopGroup with the values given.
    * @param {StopClusterGpuid} stopClusterGpuid - Ground place unique identifier of the stopCluster parent.
-   * @param {StopGroup} stopGroupInfos - StopGroup informations.
-   * @returns {StopGroup}
+   * @param {StopInfos} stopGroupInfos - StopGroup informations.
+   * @returns {void|Error}
    */
   // @ts-ignore
-  public createStopGroup(stopClusterGpuid: StopClusterGpuid, stopGroupInfos: StopGroup): void {}
+  public createStopGroup(stopClusterGpuid: StopClusterGpuid, stopGroupInfos: StopInfos): void | Error {
+    // Generate StopGroup with ground place unique identifier generator
+    const stopGroupGenerated: GroundPlaceGenerated = this.Generator.gpuid(stopGroupInfos);
+
+    // Add the StopGroup to a StopCluster
+    this.Storage.addStopGroupToGroundPlacesList(stopClusterGpuid, stopGroupGenerated);
+  }
 
   /**
    * @description Create the stopCluster with the values given.
-   * @param {StopCluster} stopClusterInfos - StopCluster informations.
-   * @returns {StopCluster}
+   * @param {StopInfos} stopClusterInfos - StopCluster informations.
+   * @returns {void|Error}
    */
-  // @ts-ignore
-  public createStopCluster(stopClusterInfos: StopCluster): void {}
+  public createStopCluster(stopClusterInfos: StopInfos): void | Error {
```

Comment:
Same

Reply:
This method now return the `StopCluster` created or an `Error` returned by the `applyGroundPlacesDiff()` method